docs/devGuide/bootcamp/exploreMarkBind.md

```diff
@@ -327,7 +331,9 @@ The `site.json` file is autogenerated and located in the root directory of the s
 
 1. Open the `site.json` file in your `mb-dev-xxx` folder.
 1. Change `"titlePrefix": "",` to `"titlePrefix": "mb-dev-xxx",`.
-1. Change `"title": "Landing Page"` to `"title": "Home"`.
+1. Under `pages`, add `"title": "Home"`.
+
+<panel src="{{baseUrl}}/userGuide/siteJsonFile.html#pages" header="**`site.json` File: `pages`**" type="info" minimized></panel>
```

Comment:
![image](https://user-images.githubusercontent.com/41845017/232308945-e333745f-13ad-420c-99c8-2532de552109.png)
https://deploy-preview-2288--markbind-master.netlify.app/devguide/development/workflow#adding-intra-site-links-to-documentation

Reply:
Updated in [786dc91](https://github.com/MarkBind/markbind/pull/2288/commits/786dc91c2eb79d973ee2866080f35e9749bf5f57).

Reply:
Sorry I think my previous reply is not accurate. It seems like somewhere below in this page I did use a panel referencing content in the userguide and it works fine. So using a panel should be ok for some reason.

Reply:
The original `panel` I included was like this:

```html
<panel src="{{baseUrl}}/userGuide/siteJsonFile.html#pages" header="**`site.json` File: `pages`**" type="info" minimized></panel>
```

Which didn't work because I was using `html` instead of `md` for the `src` - the panel you used (I think it's this one) is using `md`, which is why I think it works:

```html
<panel src="{{baseUrl}}/userGuide/deployingTheSite.md#markbind-action-tip" header="**MarkBind Action**" type="info" expanded/>
```

But using `src="{{baseUrl}}/userGuide/siteJsonFile.md#pages"` didn't display properly either, so I changed it to a normal anchor. I felt that either way as long as the developer has some convenient access to the link it shouldn't matter for this particular PR. (Of course, if a panel is better for readability, I can look into making sure the panel version works)

Regardless, I will be investigating this issue - I'll either post it as an issue or address it somewhere else (potentially as a Documentation update if this is another way to link between DG and UG).

Reply:
Interesting. I think the problem is this:
- include only works if you include a fragment that is properly enclosed (meaning you created a div/span to wrap the content)
  - you used the heading 'pages' but that is not supported (because our heading is not auto-wrapping the content below it)
- as for .md vs .html, I think needs further clarification. I see that there's one instance of .html being used in the panel docs:
![image](https://user-images.githubusercontent.com/41845017/232356618-d5b416da-e6d1-4384-a83f-d37f73258c3a.png)
but that is actually referencing a .html document. In other cases, it seems to be that panel needs to reference .md, or whatever actual type of file the src is. But this is not clear from the docs. On the flip side I vaguely remember .md and .html is interchangeable when using 'include', so we might be able to support that as well, for the special case of '.md'

Yes, please help to raise an issue for this!